config/500-controller.yaml

```diff
@@ -38,10 +38,10 @@ spec:
         - name: GH_RA_IMAGE
           value: github.com/knative/eventing-sources/cmd/github_receive_adapter
         resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
           requests:
             cpu: 100m
-            memory: 20Mi
+            memory: 1000Mi
```

Comment:
shouldn't this be 100Mi?

Reply:
@vaikas-google it's the same as the sources-controller has in `eventing` repo - happy to change, though

Reply:
@n3wscott @vaikas-google  happy to change to `100Mi` - if so... than I will also do the same for `sources_controller` in "eventing" repo (b/c that's where I did _borrow_ this value from)

thoughts?

Reply:
`1000Mi` seems like a typo.